packages/contracts-bedrock/src/L2/L1Block.sol

```diff
@@ -119,6 +129,14 @@ contract L1Block is ISemver, IGasToken {
         batcherHash = _batcherHash;
         l1FeeOverhead = _l1FeeOverhead;
         l1FeeScalar = _l1FeeScalar;
+        isDeposit = _isDeposit;
+    }
+
+    // TODO natspec
+    function setL1BlockValuesIsthmus() external {
+        isDeposit = true;
+        setL1BlockValuesEcotone(calldata);
```

Comment:
I think we should just be able to call `setL1BlockValuesEcotone` since it operates directly on the calldata, it should be an internal jump rather than a call

Reply:
We just need to make `setL1BlockValuesEcotone` public i believe, the modifier that allows it to be called both externally and internally, i get confused between public/external sometimes